main/appengine_config.py

```diff
@@ -6,6 +6,10 @@
 if os.environ.get('SERVER_SOFTWARE', '').startswith('Google App Engine'):
   sys.path.insert(0, 'lib.zip')
 else:
+  if os.name == 'nt':
+    os.name = None
+    sys.platform = ''
```

Comment:
Because in that question it was for Django and only os.name was used to check if the os was windows. In this case, Click uses both sys.platform and os.name to check if the os is Windows. Using only os.name doesn't work.

See
https://github.com/pallets/click/search?utf8=%E2%9C%93&q=sys.platform&type=Code
and
https://github.com/pallets/click/search?utf8=%E2%9C%93&q=nt&type=Code